### PR TITLE
MP-2231 ♻️ Add uuid regex

### PR DESCRIPTION
**Pull request by M4tini**
https://myparcelcombv.atlassian.net/browse/MP-2231
I already tested this fix by copying the spec to my local API.

### CODE REVIEW

File: specification/schemas/Email.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "type": "string",
+  "format": "email",
```
Comment (yoan-myparcel) [inline specification/schemas/Email.json]: If we have the pattern what value does the format bring?

File: specification/paths/SynchronizeIntegration.json [hunk 2]
```diff
@@ -33,11 +33,13 @@
                   "shop_id": {
                     "type": "string",
                     "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
```
Comment (yoan-myparcel) [inline specification/paths/SynchronizeIntegration.json]: we cannot use the Uuid schema here?

File: specification/schemas/Email.json (hunk 1 above)
Comment (M4tini) [reply specification/schemas/Email.json]: Nothing, just for clarification. Maybe they eventually update the json-api package to mention the `format` when the `pattern` does not match.

File: specification/paths/SynchronizeIntegration.json (hunk 2 above)
Comment (M4tini) [reply specification/paths/SynchronizeIntegration.json]: The problem with the Uuid schema is that it does not include an example. And we do not want the same example uuid for all our resources. That would look weird in relationship examples.

Now I think about it, we could use the Uuid schema like this:
```
"allOf": [
  {
    "$ref": "#/components/schemas/Uuid"
  },
  {
    "example": "35eddf50-1d84-47a3-8479-6bfda729cd99"
  }
]
```

**Pull request merged**